src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public Figure getRandomFigure() {
+        Random random = new Random();
+        double randomNum = random.nextInt(1, 5);
```

Comment:
Why result of random.nextInt(1, 5); is double?

Reply:
oh, that hard way to explain, easy way - fix it, and i did it. thx